identity/client/src/components/global/AppRoot.tsx

```diff
@@ -8,18 +8,17 @@
 
 import styled, { createGlobalStyle } from "styled-components";
 import { FC, ReactNode } from "react";
-import { background, bodyShort01 } from "@carbon/elements";
+import { g10, bodyShort01 } from "@carbon/elements";
 import AppHeader from "src/components/layout/AppHeader";
 import ErrorBoundary from "src/components/global/ErrorBoundary";
 
 const GlobalStyle = createGlobalStyle`
   body {
-    background: ${background};
+    background: ${g10.background};
```

Comment:
I had contrast issues while aligning the designs, so I adjusted the background globally to follow the g10 theme here instead of the previous white background.